python/lsst/cp/pipe/ptc/cpExtractPtcTask.py

```diff
@@ -809,12 +816,15 @@ def measureMeanVarCov(self, im1Area, im2Area, imStatsCtrl, mu1, mu2):
                     Covariance at (dx, dy).
                 nPix : `int`
                     Number of pixel pairs used to evaluate var and cov.
+        rowMeanVariance : `float` or `NaN`
+            Variance of the means of each row in the difference image.
+            Taken from `github.com/lsst-camera-dh/eo_pipe`.
 
             If either mu1 or m2 are NaN's, the returned value is NaN.
         """
         if np.isnan(mu1) or np.isnan(mu2):
             self.log.warning("Mean of amp in image 1 or 2 is NaN: %f, %f.", mu1, mu2)
-            return np.nan, np.nan, None
+            return np.nan, np.nan, None, np.nan
```

Comment:
The test above (L587) tests for `is None`, not `np.nan`.  Same point for the other returns below.

Reply:
Changed the test to check for `np.nan`